### ISSUE TITLE
Is the peer dependency `"prop-types": "^15.8.1"` really needed?

### ISSUE THREAD

**Issue by johannes-darms** [ts4nfdi/terminology-service-suite#9]
Is the peer dependency `"prop-types": "^15.8.1"` really needed? I thought the project uses typescript

**Comment by VincentKneip** [comment]
Elastic UI seems to need `prop-types` as peer dependency (see [README](https://github.com/nfdi4health/semlookp-widgets/blob/main/README.md?plain=1#L32-L39) or Elastic UI's [wiki docs](https://github.com/elastic/eui/tree/da50ba7f4d6a01eb9cf21a58b8ed06fe9d05861e/wiki/consuming-eui#installation)).

**Comment by jusa3** [comment]
You're right, since we don't create own React components it's not needed. Will be removed in one if the next releases. Thank you! :)
